tools/jsonschema_to_rust_dto_enum_flags.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+
+import os
+import re
+import sys
+import json
+
+from typing import Iterator, cast
+
+DEFAULT_INDENT = 4
+DOCS_URL = 'https://github.com/kamu-data/open-data-fabric/blob/master/open-data-fabric.md#{}-schema'
+PREAMBLE = """
+///////////////////////////////////////////////////////////////////////////////
+// WARNING: This file is auto-generated from Open Data Fabric Schemas
+// See: http://opendatafabric.org/
+///////////////////////////////////////////////////////////////////////////////
+
+use bitflags::bitflags;
+
+use crate::MetadataEvent;
+
+///////////////////////////////////////////////////////////////////////////////
+"""
+
+JsonType = dict[str, 'JsonType']
+SchemaName = str
+
+
+def render(schemas_dir: str) -> None:
+    print(PREAMBLE)
+
+    name, sch = read_schema(schemas_dir, 'metadata-events/MetadataEvent.json')
+
+    try:
+        print('/' * 80)
+        print(f'// {name}')
+        print('// ' + DOCS_URL.format(name.lower()))
+        print('/' * 80)
+        print()
+
+        for l in render_schema(name, sch):
+            print(l)
+        print()
+
+
```

Comment:
please remove two empty lines

(I know we should have a code auto-formatter, but I was hoping to rewrite these scripts to Rust for a long time not to invest time in Python CI)

Reply:
in Rust we would even be able to use `syn` crate to generate code ... *sigh* I wish we had an extra year to dedicate fully to the tech debt.

Reply:
> please remove two empty lines

Done

> I know we should have a code auto-formatter

the proposed code was after autoformatting, but apparently it is not as strict (PyCharm defaults) as we would like

> but I was hoping to rewrite these scripts to Rust for a long time not to invest time in Python CI)

will be great but in Python is good too

> sigh I wish we had an extra year to dedicate fully to the tech debt.

yep, our common engineering dream :)